server/routes/aws/securityhub_routes.py

```
@@ -0,0 +1,161 @@
+    if source != "aws.securityhub":
+        EVENTBRIDGE_EVENTS_FAILED.labels(reason="invalid_source").inc()
+        logger.warning(f"[SECURITY_HUB] Invalid source {source} for org {org_id}")
+        return jsonify({"error": "Invalid event source. Must be aws.securityhub"}), 400
+
+    EVENTBRIDGE_EVENTS_RECEIVED.inc()
+    logger.info(f"[SECURITY_HUB] Received valid EventBridge webhook for org {org_id}")
+
+    try:
+        # Enqueue background task to process and parse the findings
+        process_securityhub_finding.delay(payload, org_id)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)